__tests__/api.test.js

```diff
@@ -169,6 +169,64 @@ describe("POST API - insert a comment row using review_id and author FK", () =>
     })
 })
 
+describe("PATCH API - update reviews table, column votes by input", () => {
+    test("200: update votes from an object inputted", () => {
+        const dummyData = {votes: 50};
+        return request(app)
+            .patch("/api/reviews/2")
+            .send(dummyData)
+            .expect(200)
+            .then(({body}) => {
+                const expected = {
+                    status: 'Updated successfully',
+                    rowCount: 1,
+                    review: { title: 'Jenga', votes: 55 }
+                  }
+                expect(body).toEqual(expected)
+            })
+    })
+    test("400: bad input - props value", () => {
+        const dummyData = {votes: 'abc'}
+        return request(app)
+            .patch("/api/reviews/2")
+            .send(dummyData)
+            .expect(400)
+            .then(({body}) => {
+                expect(body).toEqual({msg: '400 - Bad input'})
+            })
+    })
+    test("400: bad input - incorrect prop name", () => {
+        const dummyData = {votezzz: 10}
+        return request(app)
+            .patch("/api/reviews/2")
+            .send(dummyData)
+            .expect(400)
+            .then(({body}) => {
+                expect(body).toEqual({msg: '400 - Bad input'})
+            })
+    })
+    test("400: bad input - incorrect ID in path symbol", () => {
+        const dummyData = {votes: 10}
+        return request(app)
+            .patch("/api/reviews/2+")
+            .send(dummyData)
+            .expect(400)
+            .then(({body}) => {
+                expect(body).toEqual({msg: '400 - Bad input'})
+            })
+    })
+    test("400: bad input - incorrect ID in path char", () => {
+        const dummyData = {votes: 10}
```

Comment:
This test is possibly the same as the last test ( ID is not a number ), don't forget to also include a 404 test for if the ID is a number that doesn't exist in the database